person.rb

```diff
@@ -1,3 +1,4 @@
+require './corrector'
 # Person class to identify
 class Person
   def initialize(age:, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] Great job on implementing the `Corrector` class 👍 however according to the requirements there should be an instance of `Corrector` set in the `Person` class on initialization, please refer to [this lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/composition_by_examples.md#composable-classes) for more information